react-web-app/public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <script src="https://sandbox.trustly.one/start/scripts/trustly.js?accessId={YOUR_ACCESS_ID}"></script>
+    <script src="https://sandbox.trustly.one/start/scripts/trustly.js?accessId=A48B73F694C4C8EE6306"></script>
```

Comment:
please revert this change

Reply:
As we moved the React Native to trustly-react-native-example repo, we can just ignore this PR and consider the new one. New PR https://github.com/TrustlyInc/trustly-react-native-example/pull/1